### PR TITLE
get feed function updated

### PR DESCRIPTION
**Pull request by sladyn98**

### CODE REVIEW

File: c3po/core.py [hunk 1]
```diff
@@ -52,8 +54,16 @@ def get_feed():
...
         refresh_short_token()
-    print(response.json())
-    return response.json()
+    dict = response.json()
+
+    while ('paging' in dict):
+        x = dict['paging']['next']
+        request_url = x + LTTK_GROUP_ID + '/feed'
+        response = REQ_SESSION.get(request_url, params=PAYLOAD)
+        print(dict)
```
Comment (ghostwriternr) [inline c3po/core.py]: From now on, use print statements only while debugging. Remove all print statements before pushing the code.

File: c3po/core.py [hunk 2]
```diff
@@ -52,8 +54,16 @@ def get_feed():
     response = REQ_SESSION.get(request_url, params=PAYLOAD)
     if response.status_code == 400:
         refresh_short_token()
-    print(response.json())
-    return response.json()
+    dict = response.json()
+
+    while ('paging' in dict):
+        x = dict['paging']['next']
+        request_url = x + LTTK_GROUP_ID + '/feed'
```
Comment (ghostwriternr) [inline c3po/core.py]: Pretty sure you don't need to do this. `x`is already the url for the next batch of posts in the feed. Meaning it already contains the `/feed` edge.

Comment (ghostwriternr) [reply c3po/core.py]: Also, use better variable names. Variable names must always be self-explanatory. `x` tells me nothing, but `request_url` does. It's okay to reuse variables (as long as you're doing it carefully). It should be fine to reuse `request_url` here.

**Comment by sladyn98** [comment]
Cool

**Comment by sladyn98** [comment]
So do you want Me to make changes and then send another pull request

**Comment by ghostwriternr** [comment]
> Add a new commit to the same branch when done. Changes will reflect in this PR.

**Comment by sladyn98** [comment]
I changed the function and committed the changes but I can't seem to see
any changes in the pull request

**Pull request merged**